docs/quickstarts/packet.md

```diff
@@ -5,262 +5,307 @@
 * [Introduction](#introduction)
 * [Requirements](#requirements)
 * [Step 1: Install lokoctl](#step-1-install-lokoctl)
-* [Step 2: Set up a working directory](#step-2-set-up-a-working-directory)
-* [Step 3: Set up credentials from environment variables](#step-3-set-up-credentials-from-environment-variables)
-* [Step 4: Define cluster configuration](#step-4-define-cluster-configuration)
-* [Step 5: Create Lokomotive cluster](#step-5-create-lokomotive-cluster)
+* [Step 2: Create a cluster configuration](#step-2-create-a-cluster-configuration)
+* [Step 3: Deploy the cluster](#step-3-deploy-the-cluster)
 * [Verification](#verification)
 * [Cleanup](#cleanup)
 * [Troubleshooting](#troubleshooting)
-* [Conclusion](#conclusion)
 * [Next steps](#next-steps)
 
 ## Introduction
 
-This quickstart guide walks through the steps needed to create a Lokomotive cluster on Packet with
-Flatcar Container Linux using Route53 as the DNS provider.
+This guide shows how to create a Lokomotive cluster on [Packet](https://www.packet.com/). By the
+end of this guide, you'll have a basic Lokomotive cluster running on Packet with a demo application
+deployed.
 
-By the end of this guide, you'll have a production-ready Kubernetes cluster running on Packet.
+The guide uses `t1.small.x86` as the Packet device type for all created nodes. This is also the
+default device type.
 
-## Requirements
+Lokomotive runs on top of [Flatcar Container Linux](https://www.flatcar-linux.org/). This guide
+uses the `stable` channel.
 
-* Basic understanding of Kubernetes concepts.
-* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
```

Comment:
IIUC from your comment [here][link], BGP should be enabled in the packet project. Is not listed for some reason or am I missing something?


[link]: https://github.com/kinvolk/lokomotive/pull/332#discussion_r434761881

Reply:
Thanks, fixed.